README.md

```diff
@@ -23,16 +23,15 @@ start using the CLI:
 # Currently supported features
 
 ## profiles
-* adding a new profile (--add <name> --url <URL>)
-* removing a locally registered deployment (--remove <name>)
-* list locally registered deployment (no param), 
+* adding a new profile (--create <name> --url <URL>)
+* Deleting a locally registered deployment (--delete <name>)
+* list locally registered deployment (--list), 
 * select a locally registered deployment (--select <name>)
 
 ## login
 * log into the currently selected local deployment of nexus
```

Comment:
Why local deployment ? Why not 'log into the currently selected deployment of Blue Brain Nexus' ?

Reply:
agreed.